apps/webapp/src/components/send/send-form/use-refresh-fee.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+import { sendSelector } from '../../../state/send';
+import { useStore } from '../../../state';
+
+/**
+ * Refreshes the fee in the state when the amount, recipient, selection, or memo
```

Comment:
Note that we're refreshing even when just the memo changes. Am I correct in understanding that a larger memo could result in the fee changing, since a larger memo takes up more space? Or does the fee remain the same regardless of memo?

Reply:
i believe an encrypted memo is constant size, even if it contains no memo

Reply:
OK, thanks — I've removed `memo` from the effect dependencies.